webdriver/client.py

```diff
@@ -385,12 +377,13 @@ def send_command(self, method, uri, body=None):
 
         url = urlparse.urljoin("session/%s/" % self.session_id, uri)
         response = self.send_raw_command(method, url, body)
+        value = response.body["value"]
 
-        rv = response.body["value"]
-        if not rv:
-            rv = None
+        if response.status != 200:
```

Comment:
Sure, I understand @jugglinmike's point. I guess changing send_command as you suggest might work out OK since I think we have error checking on session_id being defined elsewhere. Or we could make the `command` decorator do the url manipulation. Or we could just require the tests to use `session.transport.send` rather than wrapping it.